components/collector/src/source_collectors/grafana_k6/slow_transactions.py

```diff
@@ -30,12 +60,14 @@ def _parse_json(self, json: JSON, filename: str) -> Entities:
                 max_response_time=str(values["max"]),
                 thresholds=self._format_thresholds(metric),
             )
-            entities.append(entity)
+            if self._is_to_be_included_and_is_slow(entity):
```

Comment:
Perhaps a good improvement to move the `_is_to_be_included_and_is_slow` from parse methods to `_include_entity` methods?